central/image/service/service_impl.go

```diff
@@ -530,11 +556,24 @@ func (s *serviceImpl) EnrichLocalImageInternal(ctx context.Context, request *v1.
 
 	if !hasErrors {
 		if forceScanUpdate {
-			if err := s.enrichWithVulnerabilities(img, request); err != nil && imgExists {
-				// In case we hit an error during enriching, and the image previously existed, we will _not_ upsert it in
-				// central, since it could lead to us overriding an enriched image with a non-enriched image.
-				s.informScanWaiter(request.GetRequestId(), nil, err)
-				return nil, err
+			if err := s.enrichWithVulnerabilities(img, request); err != nil {
+				imgName := ternary.String(existingImg != nil, existingImg.GetName().GetFullName(), request.GetImageName().GetFullName())
+				log.Errorw("Enriching image with vulnerabilities",
```

Comment:
how come this is Error level?

Reply:
There was no logging in Central when vuln matching fails, a user would have to inspect Sensor logs - this is logged at error because that is what it indicates, an error occurred during enrichment.

Reply:
I misread that and thought it was `err == nil` oops